edmq_backend/src/scripts/generate_song_list.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+import shazam_data
+
+# Youtube API Request
+part = 'snippet'
```

Comment:
Curious why this is separated when it's not manipulated or reused after the single instance.

Reply:
Just to keep different data sources organized, I created a different file. Youtube api is the source of the playlist data, shazam api populates the artist and song title. I can combine if it's easier to follow or if it's cleaner to just have one script.